src/main/kotlin/org/dashevo/dpp/Extensions.kt

```diff
@@ -44,3 +45,12 @@ fun ByteArray.toBase64(): String {
 fun ByteArray.toBase64Padded(): String {
     return BaseEncoding.base64().encode(this)
 }
+
+/**
+ *
+ * @receiver ByteArray
+ * @return String The Base64 representation of this
```

Comment:
Should It say: "return the Base58 ..." ?

Reply:
yes

Reply:
Resolved here:  https://github.com/dashevo/android-dpp/pull/6/commits/9e5497090878166f83d3a69bb272d4627d4fe499